css/style.css

```diff
@@ -284,3 +284,329 @@ span {
+  padding-right: 77px;
+}
+
+.aside__display {
+  display: flex;
+  justify-content: left;
+  margin-bottom: 43px;
+}
+
+.aside__display_text {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 35px;
+  line-height: 43px;
+  margin-top: -8px;
+}
+
+.aside__display_img {
+  margin-right: 19px;
+}
+
+.aside-text__span {
+  font-size: 20px;
+  line-height: 24px;
+  font-weight: 600;
+  font-style: normal;
+}
+
+.aside__display_span {
+  display: block;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.aside__functions {
+  display: flex;
+  justify-content: left;
+  align-items: center;
+  margin-bottom: 25px;
+  padding-right: 40px;
+}
+
+.aside__functions-img {
+  display: block;
+  margin-right: 19px;
+  width: 44px;
+  height: 34px;
+}
+
+.aside__functions-text {
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 18px;
+}
+
+.aside__span {
+  display: block;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.aside__volum {
+  display: flex;
+  justify-content: left;
+  align-items: center;
+  margin-bottom: 173px;
+}
+
+.aside__volum-img {
+  height: 52px;
+  margin-left: 4px;
+  margin-right: 24px;
+  margin-top: 2px;
+}
+
+.aside__volum-text {
+  font-weight: 600;
+  font-style: normal;
+  font-size: 35px;
+  line-height: 43px;
+}
+
+.aside__volum-span {
+  display: block;
+  width: 127px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.aside__footer {
+  display: flex;
+  justify-content: space-between;
+  padding-right: 46px;
+  align-items: center;
+}
+
+.aside__footer a {
+  text-decoration: none;
+  color: #fff;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 13px;
+  line-height: 16px;
+  text-transform: uppercase;
+}
+
+.aside__footer-button {
+  border: 2px solid #fff;
+  width: 160px;
+  height: 40px;
+  padding-left: 43px;
+  padding-top: 9px;
+  transition: all .3s;
+  background-color: #e51937;
+}
+
+.aside__footer-button:hover {
+  background-color: #fff;
+  color: #e51937;
+}
+
+.aside__footer-btn {
+  display: flex;
+  align-items: center;
+  transition: all .3s;
+  justify-content: left;
+}
+
+.aside__footer-btn:hover {
+  margin-top: 3px;
+}
+
+.aside__footer-img {
+  margin-right: 8px;
+}
+
+.aside__footer-i {
+  display: inline-block;
+}
+
+.section {
+  display: grid;
+  grid-template-columns: 30.6% 69.4%;
+}
+
+.section__aside {
+  background-color: #e51937;
+  color: #fff;
+}
+
+.section__aside-title {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 30px;
+  line-height: 37px;
+  text-align: center;
+  padding-right: 15px;
+  margin-bottom: 0;
+}
+
+.section__aside-text {
+  padding-left: 56px;
+  padding-right: 47px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 18px;
+}
+
+.section__inner-info {
+  padding: 11px 21.4% 20.1% 56px;
+}
+
+.content__title {
+  font-style: normal;
+   font-weight: 600;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5948685)